src/map/mob.c

```diff
@@ -273,7 +273,7 @@ struct mob_data* mob_spawn_dataset(struct spawn_data *data) {
 	if (data->state.ai)
 		md->special_state.ai = data->state.ai;
 	if (data->state.size)
-		md->special_state.size = data->state.size;
+		md->vd->size = data->state.size;
```

Comment:
this line make the map-server.exe crashed when spawn a small/big size poring
if spawn a normal size its no problem though